static/katex.less

```diff
@@ -232,8 +232,25 @@
         & + .mop.mtight { margin-left: @thinspace; }
     }
 
+    .vlist-t {
+        display: inline-table;
+        table-layout: fixed;
```

Comment:
Apparently `table-layout: fixed` renders faster.  I'm curious if the decision to use this was motivated by perf concerns or something else.